js/js.js

```diff
@@ -0,0 +1,30 @@
+$(function(){
+	//listen for submit event and execute a function to change background
+	$("form").submit(changeBackgroundImage);
+
+
+	//function to execute the changing of the bg 
+	function changeBackgroundImage(e) {
+		e.preventDefault(); 	//prevent click event from changing
+
+
+		//variable to retrieve user input and convert to uppercase to prevent case sensitivity
+
+		var cityName = $("#city-type").val().toUpperCase();
```

Comment:
@cliffjadoo Good job on making the entered values case match your conditionals case.